arcus_quickstart.md

```diff
@@ -0,0 +1,114 @@
+<!--
+author:   Arcus Education
+email:    arcus-support@chop.edu
+version:  1.0.0
+language: en
+narrator: US English Female
+title: Arcus Quickstart
+
+link:  https://cdn.jsdelivr.net/gh/arcus/Arcus_Labs_Orientation@main/assets/styles.css
+link:  https://cdn.jsdelivr.net/gh/arcus/education_modules@main/assets/styles.css
+script: https://kit.fontawesome.com/83b2343bd4.js
+script: https://cdn.jsdelivr.net/npm/mermaid@9.4.3/dist/mermaid.min.js
+-->
+
+# Arcus Labs: Quick Start
+
+Ready to start a research project with Arcus, or have you been given a computational lab and want to know what to do next? This quick start guide will jump start your Arcus journey! 
+
+If you have questions about Arcus Labs that aren't answered here, please see our [Arcus Lab User Guide]() for more detailed documentation. 
+
+## Start a research project with Arcus
+
+To work with data provided by Arcus, you'll need an Arcus Computational Lab (called an Arcus Lab for short).
+
+When you [request an Arcus Lab](https://support.arcus.chop.edu/servicedesk/customer/portal/6/create/307), we evaluate your proposed study to make sure it's a good fit for our service, then build the computational environment and provision the data for your team so you can start work. 
+This process takes time and coordination across multiple groups within Arcus. 
+Here's what you can do to make the process as smooth as possible: 
+
+- Check first: Is your proposed study a good fit for Arcus?
+- Save time: Hone your data request beforehand
+- Know the rules: Security, privacy, and the [Arcus Terms of Use](https://arcus.chop.edu/terms-of-use).
+
+### Is your proposed study a good fit for Arcus?
+
+<div style = "background-color:white;">
+
+<script style="display: block" run-once="true" modify="false">
+mermaid.initialize({});
+
+var svg = mermaid.render(
+'arcus_fit_flowchart',
+`flowchart TB
+ accTitle: Is your proposed study a good fit for Arcus?
+ accDescr: In order for your proposed study to be a good fit for Arcus, everyone on your team must have CHOP login credentials, your proposed project must be research, your project must involve human data, and your team must be prepared to work with Arcus tools. 
+  A[Does everyone on your team\\nhave CHOP login credentials?] --Yes--> B[Is your proposed project research?];
+  A --No--> C([Arcus is likely\\nnot a good fit]);
+  B --Yes--> D[Does your project involve human data?];
+  D --Yes--> E[Is your team prepared\\nto work with Arcus tools?]
+  E --Yes---> F([Your project is likely\\na good fit for Arcus!]);
+  B --No--> C;
+  D --No--> C;
+  E --No--> C;
+`,
+function(g) {
+    return true;
+})
+
+"HTML: " + svg
+</script>
+
+</div>
+
+Let's go into a bit more detail about each of these questions. 
+
+#### Does everyone on your team have CHOP login credentials?
+Anyone who will need access to the data has to have CHOP credentials. 
+You will not be able to share the data with outside collaborators unless they first get NTP status with CHOP.
```

Comment:
Maybe link to where PIs can learn more about NTP requests?  Also mention (I think this is true) that you have to specify that NTPs get email addresses / CHOP logins.  I suspect there's a good Forum post about this (whose links probably need checking at this point)